kickstarter/android-oss
# NT-1689: Update greys/blacks

# 📲 What

```
- Instances of white (#FFFFFF) are replaced with KDS-white (#FFFFFF)
- Instances of black (#000000) are replaced with KDS-black (#000000)
- KDS-support-100 (#F3F3F3) is added replaces ksr-grey-100 and ksr-grey-200
- KDS-support-300 (#D1D1D1) is added replaces ksr-grey-300, ksr-grey-400, and ksr-grey-500
- KDS-support-400 (#696969) is added replaces ksr-dark-grey-400 and ksr-dark-grey-500
- KDS-support-500 (#464646) is added
- KDS-support-700 (#222222) is added replaces ksr-soft-black
```

# 👀 See
<img width="566" alt="Screen Shot 2020-12-08 at 11 30 26 AM" src="https://user-images.githubusercontent.com/4083656/101532204-0f7a4800-3949-11eb-8a50-5b4eca82ee68.png">

|  |  |

# 📋 QA
Make sure the basic functionalities looks fine
- Load projects
- Log in
- Pledge to some project
- Watch your messages
- Look your pledge to a previous project
- Make a search

# Story 📖

[NT-1689](https://kickstarter.atlassian.net/browse/NT-1689)
